superset-frontend/src/explore/components/OptionControls.tsx

```diff
@@ -84,7 +84,7 @@ export const HeaderContainer = styled.div`
 export const LabelsContainer = styled.div`
   padding: ${({ theme }) => theme.gridUnit}px;
   border: solid 1px ${({ theme }) => theme.colors.grayscale.light2};
-  border-radius: 3px;
+  border-radius: 4px;
```

Comment:
Could we maybe use `gridUnit` here as well? As it will have the same result but keep it consistent with the rest of the styling?
```suggestion
  border-radius: ${({ theme }) => theme.gridUnit}px;
```